tests/phpunit/CRM/Contract/PaymentAdapter/AdyenTest.php

```diff
@@ -308,10 +307,11 @@ public function testPauseAndResume() {
 
     $recurContribQuery = Api4\ContributionRecur::get(FALSE)
       ->addSelect('contribution_status_id:name')
+      ->addWhere('id', $recurId)
```

Comment:
This test was trolling me ... looks like on a "clean" test environment, there's already an existing `ContributionRecur` with status=Pending (idk why), and we were fetching that one because the `addWhere()` was missing.